src/Umbraco.Cms.Api.Management/Mapping/Dictionary/DictionaryMapDefinition.cs

```diff
@@ -37,9 +37,14 @@ private void Map(UpdateDictionaryItemRequestModel source, IDictionaryItem target
         target.DeleteDate = null;
     }
 
-    // Umbraco.Code.MapAll -Id -Key -CreateDate -UpdateDate -Translations
+    // Umbraco.Code.MapAll -Id -CreateDate -UpdateDate -Translations
     private void Map(CreateDictionaryItemRequestModel source, IDictionaryItem target, MapperContext context)
     {
+        if (source.Id != null)
```

Comment:
Pattern matching is the new hotness 🔥 

```suggestion
        if (source.Id is not null)
```